doc/reference/bluetooth/hci.txt

```diff
@@ -800,6 +800,140 @@ When the Set_Scan_Request_Reports command has completed, a Command Complete
 event shall be generated.
 
 
+Zephyr Write Tx Power Level (per Role/Connection) Command
+=========================================================
+
+This command dynamically modifies BLE Tx power level given a Role and  or
+Connection handle.
+
++--------------------------+-------+--------------------+--------------------+
+| Command                  | OCF   | Command            | Return             |
+|                          |       | Parameters         | Parameters         |
++--------------------------+-------+--------------------+--------------------+
+| Write_Tx_Power_Level     | 0x00E | LL_State,          | Status,            |
+|                          |       | Handle,            | LL_State,          |
+|                          |       | Tx Power Level     | Handle             |
++--------------------------+-------+--------------------+--------------------+
```

Comment:
Add Tx Power Level as a return parameter. If the controller does not support the requested TX power, then the actual one will be in the Command Complete return value. For example, if you request -50dBm on a Nordic chip, the closest one supported is actually -40dBm, which would be what you return in the Command Complete.
Please also document that the controller will set the TX power to the closest supported level.

Reply:
See the  `7.8.53 LE Set Extended Advertising Parameters` in the spec and follow the format there. Please also split the parameters in 2 sections:

**Command parameters:**
and
**Return parameters**

just like the spec does. Name the return parameter as `Selected_Tx_Power` just like the spec does.

Reply:
In general I have no problem following the spec convention and using the separation

**Command parameters:**
and 
**Return parameters:**

but then I will completely brake the formatting conventions of this entire file - see for instance the [Zephyr Set Trace Enable Command](https://github.com/zephyrproject-rtos/zephyr/pull/19779/files/32c04d9968f3c5cce18cad74a4cdb633c0a701ce..de505b96597559c418fbdf6def1fb7b6b91b3e68#diff-9c37bb9bf44e4a2035f47bde14315da6R447) and alike.

So I'd rather keep the original formatting of this file and if-need-be the Command/Return separation could be handled in a separate commit. The naming conventions you and @joerchan proposed will be of course addressed.

Reply:
> but then I will completely brake the formatting conventions of this entire file - see for instance the Zephyr Set Trace Enable Command and alike.

Fair point. Follow the current convention. We can update to the spec convention for all commands as a separate fix.

Reply:
Added the `Selected_Tx_Power` [return parameter](https://github.com/zephyrproject-rtos/zephyr/pull/19779/commits/b8c7a15362e46dbcc1238e7bf79694779119a062#diff-d32c7d39d131dcc56d85c8b8b7d5b328R148) which is yielded by each radio upon its Tx power flooring operation. The [documentation](https://github.com/zephyrproject-rtos/zephyr/pull/19779/commits/6012d2b2ca1f5515ee0ae6dfc760301bfc4b0faa#diff-9c37bb9bf44e4a2035f47bde14315da6R908) reflects that as well.